src/mon/Monitor.cc

```diff
@@ -5708,7 +5708,12 @@ bool Monitor::_scrub(ScrubResult *r,
 
     bufferlist bl;
     int err = store->get(k.first, k.second, bl);
```

Comment:
When store is RocksDB, the only possible err are 0 and -ENOENT.
All other errors are handled with `ceph_abort_msg(s.getState());` that prints RocksDB internal failure reason.

I guess this is valuable extension; we will see that the reason was -ENOENT.

Reply:
Yep, so we can still use cpp_strerror(err) here, right?